src/pages/dashboard/e2e.dashboard.spec.ts

```diff
@@ -51,37 +57,261 @@ describe("Dashboard", () => {
+
+    expect(await page.$$("#burger-menu")).toBeDefined();
+
+    await page.click("#burger-menu");
+    await page.waitForSelector("#close-header-menu");
+    expect(await page.$$("#close-header-menu")).toBeDefined();
+    expect(await page.$$("#tablet-or-mobile-menu")).toBeDefined();
+    await page.click("#close-header-menu");
+    expect(!(await page.$("#tablet-or-mobile-menu")));
+  });
+
+  it("Should have items in header menu", async () => {
+    await page.waitForSelector("#burger-menu");
+    await page.click("#burger-menu");
+    await page.waitForSelector("#header-links");
+
+    const linksParent = await page.$("#header-links");
+    expect(
+      await linksParent.$$eval("a", (nodes) => nodes.map((n) => n.innerText))
+    ).toEqual(["Home", "Viewer", "Debug", "GitHub"]);
+
+    await page.waitForSelector("#compare-tablet-or-mobile-button");
+    const compareText = await page.$eval("#compare-tablet-or-mobile-button", (e) => e.textContent);
+    expect(compareText).toContain("Compare");
+
+    await page.waitForSelector("#help-button-tablet-or-mobile");
+    const helpText = await page.$eval("#help-button-tablet-or-mobile", (e) => e.textContent);
+    expect(helpText).toContain("Help");
+
+    await page.click("#close-header-menu");
+    await page.waitForSelector("#burger-menu");
+  }, 30000);
+
+  it("Contains dashboard canvas", async () => {
+    await page.waitForSelector("#dashboard-app");
+
+    const dashboardCanvas = await page.$$("#dashboard-app");
+    expect(dashboardCanvas).toBeDefined();
+  });
+
+  it("Should go to the Viewer page", async () => {
     await page.goto("http://localhost:3000");
+    await page.waitForSelector("#burger-menu");
+    await page.click("#burger-menu");
     await page.waitForSelector("#header-links");
-    await page.click("a[href='/comparison']");
+    await page.click("a[href='/viewer']");
+
     const currentUrl = page.url();
-    expect(currentUrl).toBe("http://localhost:3000/comparison");
+    expect(currentUrl).toBe("http://localhost:3000/viewer?tileset=san-francisco-v1.7");
+    const controlPanel = await page.$$("#control-panel");
+    expect(controlPanel).toBeDefined();
+    expect(!(await page.$("#tablet-or-mobile-menu")));
+  });
+
+  it("Should go to the Debug page", async () => {
+    await page.goto("http://localhost:3000");
+    await page.waitForSelector("#burger-menu");
+    await page.click("#burger-menu");
+    await page.waitForSelector("#header-links");
+    await page.click("a[href='/debug']");
+    await page.waitForTimeout(5000);
+
+    const currentUrl = page.url();
+    expect(currentUrl).toBe("http://localhost:3000/debug?tileset=san-francisco-v1.7");
+    const toolBar = await page.$$("#tool-bar");
+    expect(toolBar).toBeDefined();
+    expect(!(await page.$("#tablet-or-mobile-menu")));
   });
 
+  // TODO turn it on back then compare dropdown menu will be implemented
+
+  // it("Should go to the Comparison page", async () => {
```

Comment:
As I can see, dropdown in the mobile mode does work.

Reply:
This one as well will be removed here - https://github.com/visgl/loaders.gl-showcases/pull/75